app/controllers/application_controller.rb

```diff
@@ -1,5 +1,13 @@
 class ApplicationController < ActionController::Base
   def current_user
```

Comment:
-[ ] Great job with your ApplicationController. Your code still includes a `current_user` method in the `ApplicationController`, which contradicts the project requirements. The instruction explicitly states that the `current_user` method should be removed, as Devise already provides a method for this purpose. Please refactor your code by eliminating the `current_user` method from the `ApplicationController`. Ensure that you are relying on Devise's built-in functionality for managing the current user. If you encounter any issues or need guidance, feel free to ask for assistance.